gk/main.c

```diff
@@ -634,6 +708,14 @@ gk_proc(void *arg)
 			process_gk_cmd(gk_cmds[i], instance);
 			mb_free_entry(&instance->mb, gk_cmds[i]);
         	}
+
+		if (acl.num > 0) {
+			ret = process_ipv6_acl(&gk_conf->net->front,
+				lcore, &acl);
+			if (ret < 0)
+				RTE_LOG(ERR, GATEKEEPER,
+					"gk: can't classify IPv6 packets\n");
+		}
```

Comment:
1. In order to make this block even less intrusive for the blocks adding support for the IPv6 filter, put the whole block into an inline function. Being inline is important to avoid the cost of calling the new function and return if `acl.num` is not greater than zero.

2. `process_ipv6_acl()` should absorb the `RTE_LOG()` into its body.

```